ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -344,6 +350,106 @@ protected static void endToEndTest(Port targetPort, String routerDatacenter, Str
       stream = channel.receive().getInputStream();
       GetResponse resp5 = GetResponse.readFrom(new DataInputStream(stream), clusterMap);
       assertEquals(ServerErrorCode.Blob_Not_Found, resp5.getPartitionResponseInfoList().get(0).getErrorCode());
+
+      System.out.println("Begin to stop a BlobStore");
+      // stop the store via AdminRequest
+      AdminRequest adminRequest =
+          new AdminRequest(AdminRequestOrResponseType.BlobStoreControl, partitionIds.get(0), 1, "clientid2");
+      BlobStoreControlAdminRequest controlRequest = new BlobStoreControlAdminRequest((short) 3, false, adminRequest);
```

Comment:
will 0 work here (instead of 3)? - I want to avoid unpredictable failures due to thread scheduling